### PR TITLE
feat: eastern mediterranean malaria repo landing page

### PR DESCRIPTION
**Pull request by deeonwuli**
### :pushpin: References

-   **Issue:** Closes https://app.clickup.com/t/345yjdn

### :art: Screenshots
Screenshot 2022-07-27 at 14 37 50
Screenshot 2022-07-27 at 14 38 16

### CODE REVIEW

File: src/domain/models/east_mediterranian_mal/Malaria.jsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import _ from "lodash";
+import i18n from "../../../locales";
+
+const MalariaRepoDescription = () => {
+    return (
+        <>
+            <p>{i18n.t("Malaria endemic countries in the region are invited to upload a subset of their malaria related data (by district and month) into this DHIS2 instance.")}</p>
+            <p>{i18n.t("Some countries are already implementing DHIS2 for their routine HMIS and have already adopted the malaria modules. These countries can ready link the two systems. Others are using other HMIS systems and will need to use the Bulk Load app to load the data.")}</p>
+            <p style={{ textAlign: "left" }}>{i18n.t("This repository will be useful for the following:")}</p>
```
Comment (tokland) [inline src/domain/models/east_mediterranian_mal/Malaria.jsx:9]: Move to styles object or styled-components or useStyles, whatever the rest of the app is using.

**Review by tokland — APPROVED** [review]
Code-wise, the PR looks good, I just added some a comment about the styles, but's it's very minor.

**Review by ifoche — CHANGES_REQUESTED** [review]
From a functional perspective, at least for me I'm not being redirected to the right place when I click on any link. I'm basically taken to the landing page again... @deeonwuli is it working for you?

**Comment by deeonwuli** [comment]
> From a functional perspective, at least for me I'm not being redirected to the right place when I click on any link. I'm basically taken to the landing page again... @deeonwuli is it working for you?

it wasn't working but i just pushed a fix and only the Data Quality Dashboard isn't working now seeing as we still don't have a link for that @ifoche

**Review by ifoche — CHANGES_REQUESTED** [review]
Thank you @deeonwuli . From a functional perspective, now the links take to the appropriate place (but from the data quality dashboard, that the client yet has to provide it). 

The client provided the group that this landing page should be exposed to. It is a group that doesn't exist in the instance you're testing, but could you please make the change already?

User Group Name: MAL_EMRO Repository
User Group ID: FpQ7a5OylZH

screenshots_1661246345_1119_23082022_599x346

Thank you!

**Review by ifoche — CHANGES_REQUESTED** [review]
Just realized that while the EMRO WHO logo has been updated when you're inside the landing pag
screenshots_1661258658_1444_23082022_1766x340
e...


however, from outside, when you're looking at all the available landing pages, the old logo remains there...
screenshots_1661258495_1441_23082022_1317x782

Could you please change the icon there too?

**Comment by deeonwuli** [comment]
> Just realized that while the EMRO WHO logo has been updated when you're inside the landing page... screenshots_1661258658_1444_23082022_1766x340
> 
> however, from outside, when you're looking at all the available landing pages, the old logo remains there... screenshots_1661258495_1441_23082022_1317x782
> 
> Could you please change the icon there too?

@ifoche when i change the icon in the available landing pages to the new icon, it doesn't show because it's the same white colour as the background.

**Comment by ifoche** [comment]
I see, let me ask Ray to create a blue-foreground version of that logo and provide it to you in the clickup ticket

**Review by ifoche — CHANGES_REQUESTED** [review]
thanks for the logo change @deeonwuli . I think we're almost there. Now if you could take a look to the comments from Ray in the ticket and try to implement some of the changes... it doesn't need to be all of them, but at least some of them so the aspect of the landing is more appealing. 

In particular I think that what's about the content is something that the client has established so we cannot change it before talking to him (I have a meeting with him on Friday). But for instance, it looks like the welcome text title color change plus the redistribution of the bullet points could be a huge improvement, plus some padding between the 5 options below, only those things would already do a big change. 

In addition, if you could also add the following description to the WHO Data Quality App:
WHO Data Quality Tool for DHIS2, based on the WHO Data Quality Framework.

I think at least there is something there, even if it is not same length of the other entries

**Pull request merged**